src/lib.rs

```diff
@@ -0,0 +1,220 @@
+extern crate regex;
+
+use regex::Regex;
+
+#[macro_use]
```

Comment:
I don't think this line makes sense -- I've only ever seen a `#[macro_use]` before an `extern crate` declaration, and AFAIK there are no macros defined in the stdlib that you have to explicitly import.